tests/common/dualtor/dual_tor_utils.py

```diff
@@ -1410,3 +1413,27 @@ def is_tunnel_qos_remap_enabled(duthost):
         return False
     return "enabled" == tunnel_qos_remap_status
 
+@pytest.fixture(scope="session")
+def config_dualtor_arp_responder(tbinfo, duthost, mux_config, ptfhost):
+    """
+    Apply standard ARP responder for dualtor testbeds
+
+    In this case, ARP responder will reply to ARP requests and NA messages for the
+    server IPs configured in the ToR's config DB MUX_CABLE table
+    """
+    ptfhost.copy(src=os.path.join(SCRIPTS_SRC_DIR, ARP_RESPONDER_PY), dest=OPT_DIR)
+    arp_responder_conf = {}
+    tor_to_ptf_intf_map = duthost.get_extended_minigraph_facts(tbinfo)['minigraph_ptf_indices']
+
+    for tor_intf, config_vals in mux_config.items():
+        ptf_intf = "eth{}".format(tor_to_ptf_intf_map[tor_intf])
+        arp_responder_conf[ptf_intf] = [
+            str(ipaddress.ip_interface(config_vals["SERVER"]["IPv4"]).ip),
+            str(ipaddress.ip_interface(config_vals["SERVER"]["IPv6"]).ip)]
+
+    ptfhost.copy(content=json.dumps(arp_responder_conf, indent=4, sort_keys=True), dest="/tmp/from_t1.json")
+    ptfhost.host.options["variable_manager"].extra_vars.update({"arp_responder_args": ""})
+    ptfhost.template(src="templates/arp_responder.conf.j2", dest="/etc/supervisor/conf.d/arp_responder.conf")
+
+    supervisor_cmd = "supervisorctl reread && supervisorctl update && supervisorctl restart arp_responder"
+    ptfhost.shell(supervisor_cmd)
```

Comment:
Do we need to stop the `arp_responder` after the testcases?

Reply:
Good call, pushed a new commit to stop it during teardown